src/component/notification.js

```diff
@@ -54,12 +53,6 @@ export const NotificationBar = ({ notification, display, style }) =>
           <Alert type={notification.type} style={barStyles.alert} />
           <Text style={barStyles.text}>{notification.message}</Text>
         </View>
-        {notification.handler ? (
```

Comment:
> I thought these handlers were pretty helpful, e.g. [here](https://github.com/lightninglabs/lightning-app/blob/master/src/action/transaction.js#L156).

The handler buttons are still used in the notifications list view under setting. Unfortunately the react navigation stack gets completely confused if the user is able to navigate to the log screen from any other view. This leads to all sorts of side effects and I didn't see any other way to resolve this issue without a larger refactoring.

> Are we removing them because they don't work on mobile? If so, maybe it's worth keeping them on desktop.

We can keep them on the desktop. But I'd like to move to react navigation there as well. So we'll likely run into similar stack based routing issues.

Reply:
Any chance of letting go of `react-navigation` and instead transitioning mobile to a more desktop-like navigator? Seems like `react-navigation` doesn't give us much for free and is causing us to remove a helpful feature.

If we're set on using `react-navigation`, could we remove all the `handler` logic completely? there's a lot of vestigial traces right now, e.g. [here](https://github.com/lightninglabs/lightning-app/blob/master/src/action/transaction.js#L156).

Reply:
> Any chance of letting go of `react-navigation` and instead transitioning mobile to a more desktop-like navigator? Seems like `react-navigation` doesn't give us much for free and is causing us to remove a helpful feature.

Navigation is a hard problem. We don't want to solve that ourselves. There have been multiple libraries by AirBnb and Facebook that were adopted and replaced again by the community. Unfortunately `react-navigation` is the best we have right now on mobile.

It's also worth noting that we aren't using it to its full potential yet. This would required a refactoring of passing state through the nav apis instead of props. Perhaps we'll take that route once we use it on all platforms. But that would be significant coupling of our architecture to `react-navigation`. I'd like to explore and better understand desktop integration before going that route though.

> If we're set on using `react-navigation`, could we remove all the `handler` logic completely? there's a lot of vestigial traces right now, e.g. [here](https://github.com/lightninglabs/lightning-app/blob/master/src/action/transaction.js#L156).

I've changed the PR so that the error handler is still visible on the desktop. That way we work around the current limitations on mobile, while keeping the current functionality on the desktop.

Reply:
> Navigation is a hard problem. We don't want to solve that ourselves.

Got it, sounds good to me!

> I've changed the PR so that the error handler is still visible on the desktop.

Good compromise :)